src/main/java/org/primefaces/component/datatable/export/DataTableExcelExporter.java

```diff
@@ -227,6 +243,35 @@ protected Sheet createSheet(Workbook wb, String sheetName, ExcelOptions options)
         return wb.createSheet(sheetName);
     }
 
+
+    protected String getMonitorKeyCookieName(FacesContext context, ELContext elContext) {
+        String monitorKeyCookieName = Constants.DOWNLOAD_COOKIE + context.getViewRoot().getViewId().replace('/', '_');
```

Comment:
I wonder if this could be moved to ResourceUtils as a static method and reused between FileDownload and this?

Reply:
done